FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Microsoft.Health.Fhir.Synapse.Core.csproj

```diff
@@ -20,6 +20,7 @@
     <ProjectReference Include="..\Microsoft.Health.Fhir.Synapse.Common\Microsoft.Health.Fhir.Synapse.Common.csproj" />
     <ProjectReference Include="..\Microsoft.Health.Fhir.Synapse.DataClient\Microsoft.Health.Fhir.Synapse.DataClient.csproj" />
     <ProjectReference Include="..\Microsoft.Health.Fhir.Synapse.DataWriter\Microsoft.Health.Fhir.Synapse.DataWriter.csproj" />
+    <ProjectReference Include="..\Microsoft.Health.Fhir.Synapse.HealthCheker\Microsoft.Health.Fhir.Synapse.HealthCheck.csproj" />
```

Comment:
the project folder is also needed to refine to `HealthChecker`

Reply:
Thanks, fixed it.